catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -267,19 +268,37 @@ public String getResponseWrapper(String responseType, Object response) {
       attributeProperties.put("type", descriptor.getType().getAttributeFormat().name());
       attributeProperties.put("multivalued", descriptor.isMultiValued());
       attributeProperties.put("id", descriptor.getName());
+      attributeProperties.put("isInjected", true);
```

Comment:
Do you think these strings should be pulled into constants?

Reply:
i'll pull them out